src/main/java/org/spongepowered/api/item/inventory/type/Interactable.java

```diff
@@ -26,12 +26,45 @@
 
 import org.spongepowered.api.entity.living.player.Player;
 import org.spongepowered.api.item.inventory.Inventory;
+import org.spongepowered.api.plugin.PluginContainer;
+
+import java.util.Set;
 
 /**
  * Interface for inventories which may be interacted with by Players.
  */
 public interface Interactable extends Inventory {
 
+    /**
+     * Gets the current viewers looking at this Inventory.
+     *
+     * @return The current viewers of this inventory
+     */
+    Set<Player> getViewers();
+
+    /**
+     * Checks for whether this Inventory currently has viewers.
+     *
+     * @return True if viewers are currently looking at this inventory
+     */
+    boolean hasViewers();
+
+    /**
+     * Shows this Inventory to the given viewer.
+     *
+     * @param viewer The viewer to show this inventory to
+     * @throws IllegalArgumentException if a {@link PluginContainer} is not the root of the cause
+     */
+    void open(Player viewer) throws IllegalArgumentException;
```

Comment:
I would get rid of these and force devs to use `Player#openInventory` and `Player#closeInventory` instead.

Reply:
it really just redirects to exactly that

Reply:
Then there is no need to keep them?